tests/base_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+class BaseTest:
+
+    def login_new_user(self, mozwebqa, user):
```

Comment:
@davehunt I implemented this change you suggested and I do think I like it better. You can see it in the latest commit that I just pushed.